lib/cloudinary/railtie.rb

```diff
@@ -3,7 +3,9 @@ class Cloudinary::Railtie < Rails::Railtie
     Dir[File.join(File.dirname(__FILE__),'../tasks/**/*.rake')].each { |f| load f }
   end
   config.after_initialize do |app|

```

Comment:
I believe the `after_initialize` is not needed anymore

Reply:
I agree with you, this is redundant with the "on load" hook, but does not harm IMHO.

I kept the changes minimal so that it would be easier for the change to be accepted.